lib/BackendClient.js

```diff
@@ -100,6 +99,12 @@ var BackendClient = (function(){
             return getLiveEntries();
         }
     };
+    BackendClient.SetLastSessionTime=function(TimeTOSet){
```

Comment:
1. use camelCased function and argument names - "setLastSessionTime", "timeToSet", etc.
2. since these are basically private, use _ as the name prefix (e.g. _setLastSessionTime)
3. There's no real gain from the getter\setter method - I'd go with the variable itself or define a property (https://msdn.microsoft.com/en-us/library/dd548687(v=vs.94).aspx)